spec/requests/sidenav_pages_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe "Side Navbar" do
+  subject { page }
+
+  describe "Self Assessment Link" do
+    let!(:user) { FactoryGirl.create(:user) }
+    let!(:ac) { FactoryGirl.create(:associate_consultant,
+      user: user) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 9, 19 should be on its own line.